spi/core-spi/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+val jacksonVersion: String by project
+val okHttpVersion: String by project
+val jodahFailsafeVersion: String by project
+
+
+plugins {
+    `java-library`
+    `maven-publish`
+}
+
+dependencies {
+    api("com.fasterxml.jackson.core:jackson-core:${jacksonVersion}")
+    api("com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}")
+    api("com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}")
+    api("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVersion}")
+    api("com.squareup.okhttp3:okhttp:${okHttpVersion}")
```

Comment:
okhttp and failsafe dependencies should be defined in `core-base` and not in "core-spi"

Reply:
A lot of extensions depend on Failsafe and OkHttp. If we move those into `core:base` this would mean, that these extensions require a dependency on `core:base` in addition to `core-spi`.
Just so we're aware of that.

Reply:
I think this is correct... without the `core:base` dependency, the `OkHttpClient` and the `RetryPolicy` services cannot be obtained